delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/utils/JsonUtils.scala

```diff
@@ -64,6 +64,24 @@ trait JsonUtils {
   def remoteAllKeys(json: Json, keys: String*): Json =
     removeNested(json, keys.map(k => (kk => kk == k, _ => true)))
 
+  /**
+    * Replace in the passed ''json'' the found key value pairs in ''from'' with the value in ''toValue''
+    */
+  def replace(json: Json, from: (String, Json), toValue: Json): Json = {
+    def inner(obj: JsonObject): JsonObject =
```

Comment:
`val (fromKey, fromValue) = from` would make the pattern match below more readable